### PR TITLE
lint. it does a body good.

### PR DESCRIPTION
**Pull request by zburke**
@JohnC-80 , I'm not confident about changing `& .root` to `&.root` in `TextLink`, but I can't get stylelint to be quiet otherwise. Moving that block above the main `.root` block causes even more complaints 😕 .

### CODE REVIEW

File: lib/TextLink/TextLink.css [hunk 1]
```diff
@@ -79,8 +79,7 @@
 }
 
 div[class*="Selected"] {
-
-  & .root {
+  &.root {
```
Comment (JohnC-80) [inline lib/TextLink/TextLink.css:82]: Yeah... that's definitely not what we want here....
`div[class*="Selected"].root` = both classes on the same element vs
`div[class*="Selected"] .root` = the `.root` class within the `.*Selected*` element. I'll fuss around and see if I can appease it.

Comment (JohnC-80) [reply lib/TextLink/TextLink.css:82]: Stylelint is being a bear about this. So `:not` is practically the king of all pseudo selectors - it has high specificity, so SL's going to nag about those selectors as long as anything comes after this. @zburke, I recommend `/* stylelint-disable */` here.

**Review by JohnC-80 — CHANGES_REQUESTED** [review]

File: lib/TextLink/TextLink.css (hunk 1 above)
Comment (JohnC-80) [reply lib/TextLink/TextLink.css:82]: Only way I think we might appease is if we just forgot about the nice nesting postcss allows for us and wrote out the raw selectors old-school style.

**Comment by zburke** [comment]
`linter-thingy-disable` should be `linter-thingy-stfu`. It would be good for developers' mental health.

**Review by JohnC-80 — APPROVED** [review]

**Pull request merged**